core/tests/protractor_utils/forms.js

```diff
@@ -385,7 +385,10 @@ var RichTextChecker = function(arrayOfElems, arrayOfTexts, fullText) {
 
   var _readFormattedText = function(text, tagName) {
     expect(arrayOfElems[arrayPointer].getTagName()).toBe(tagName);
-    expect(arrayOfElems[arrayPointer].getInnerHtml()).toBe(text);
+    expect(
+      browser.executeScript(
```

Comment:
I have tried the method you described. It seems to work fine, will wait and see what Travis says.